.github/github-bot.yml

```diff
@@ -19,4 +19,10 @@ github-team:
 prchecklist:
   title: Pull Request Checklist
   checklist:
-    - 'Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?'
+    - 'Updated the [documentation](https://status.im/build_status/), if affected'
+    - 'Added or updated inline comments explaining intention of the code'
+    - 'Ensured that all new UI elements have been assigned accessibility IDs'
+    - 'E2E tests for new/changed app behavior have been written'
+    - 'Briefly described what was tested and what platforms were used'
+    - 'In case of UI changes, ensured that UI matches Figma'
+    - 'In case of UI changes, requested review from designer on my team'
```

Comment:
What is "my team"?

Reply:
Hmm, maybe not actually including the working GH handle, otherwise he'll be flooded with notifications.